modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/impl/TestStorageEngineTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.storage.impl;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.is;
+
+import org.apache.ignite.internal.testframework.BaseIgniteAbstractTest;
+import org.junit.jupiter.api.Test;
+
+class TestStorageEngineTest extends BaseIgniteAbstractTest {
```

Comment:
What if we extend it from `org.apache.ignite.internal.storage.engine.AbstractStorageEngineTest`? Maybe it won't work, I don't know. Have you tried?

Also, maybe this test should be in base class, for all engines at once

Reply:
Changed structure of storage engine tests: split them to persistent and volatile engines; the latter now contain the test you pointed at

Reply:
Cool, thank you!